tests/transactional/install_updates.pm

```diff
@@ -34,12 +34,21 @@ sub run {
     }
 
     # First we update the system
+    # By default we always use zypper patch
+    # Optionally we can also do a zypper up before
+    if (get_var('TRANSACTIONAL_UPDATE_UP')) {
+        record_info('UPDATE', 'Updating system');
+        trup_call('up', timeout => 1800);
+        process_reboot(trigger => 1);
+    }
+    record_info('PATCH', 'Patching system');
```

Comment:
Is  it correct that in case `TRANSACTIONAL_UPDATE_UP` i would do:
- zypper up
- fully patch
- add repo
- fully patch

Shouldn't it be ? 
- zypper up or  fully patch
- add repo
- fully patch

Reply:
Let's go with 3rd approach. Thanks!